src/app/loginComponents/accounts/external/accounts.component.scss

```diff
@@ -0,0 +1,45 @@
+@use '~@angular/material' as mat;
+@import '/src/materialColorScheme.scss';
+
+.account-logo {
+  width: 3.5em;
+  margin-right: 1em;
+}
+
+.token-form-field {
+  width: 300px;
+  font-size: 12px;
+}
+
+.color {
+  color: mat.get-color-from-palette($dockstore-app-accent-1, darker);
+}
+
+.mat-badge-medium.mat-badge-above .mat-badge-content {
+  top: 0px;
+  left: 40px;
+  background: #26bfa5;
```

Comment:
this is the same color as:
```
.workflow-tab-group.mat-primary .mat-ink-bar {
  background: #26bfa5 !important;
}

```

Make a constant for them in the styles.scss or something and name it something descriptive